trillian/log_server/trampoline.go

```diff
@@ -0,0 +1,42 @@
+package main
```

Comment:
This is a "trampoline" that takes arguments from `argv` and a specified JSON file and passes them to the actual binary. We can't use environment variables to pass stuff into Docker (because we're using `FROM scratch`, we can't rely on shell env var expansion inside the ENTRYPOINT or RUN directives), so we have this workaround.